src/cli/cmd/resolve.rs

```diff
@@ -16,6 +16,10 @@ pub(crate) struct ResolveSubcommand {
     #[arg(long)]
     json: bool,
 
+    /// Build the resolved path with Nix.
+    #[arg(short, long, default_value_t = false)]
+    build: bool,
```

Comment:
This isn't even correct. Since you pass `--max-jobs 0`, no building will happen. It'll either substitute the path (if it exists), or fail. Maybe `--fetch` or `--realise` (British spelling on purpose to match the rest of the Nix CLI) would be a better flag.

Reply:
Switched to `--fetch` but I'm open to suggestion